openshift-knative-operator/cmd/operator/kodata/ingress/0.21/1-200-clusterrole.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  # These are the permissions needed by the Istio Ingress implementation.
+  name: knative-serving-istio
+  labels:
+    serving.knative.dev/release: devel
```

Comment:
Is this a temporary label used to identify these new objects or does is it evaluated in code, too ?

Reply:
I should've noted that everything under `kodata` in this PR is automatically propagated from net-istio. These are literally straight copies from the net-istio repo. We might want to fix this label though, though it prolly doesn't do any harm to leave it like this either.

Reply:
Just out of curiosity: This label is updated during a release, right ? (so it's e.g. `"0.21"` for the 0.21 release) and then reflected here as well when we are doing our release that depends on a released version of Knative.

If so, it makes totally sense (but I would probably call it `nightly` or so, or even better like in Maven land `0.23-SNAPSHOT` ....

But I agree that this should be fixed upstream, not here.

Reply:
Yeah, usually it's updated during a release. In fact, the other manifests pull from released versions of Knative and we could do that here too. We'd only have to implement a YAML filter then because we don't want the entire manifest of net-istio but leave out the Gateways for example.